x/oracle/proposal_handler.go

```diff
@@ -0,0 +1,33 @@
+package oracle
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/medibloc/panacea-core/v2/x/oracle/keeper"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+)
+
+func NewOracleProposalHandler(k keeper.Keeper) govtypes.Handler {
```

Comment:
This handler is executed once when the first proposal is registered, and then again when the proposal is passed.

When the first proposal is registered, it is stored in the cache, so it is not stored in the kvStore.
After the proposal is passed, it is stored in the kvStore.